### ISSUE TITLE
Connecting two different controllers to OpenEmu

### ISSUE THREAD

**Issue by gcspoon** [OpenEmu/OpenEmu#4824]
Hi, I'm new to OpenEmu and I love it so far!

However, I'm having trouble getting two Bluetooth controllers to work with OpenEmu (I want to play NHL '95 for Sega Genesis in two player mode).

I was able to connect the 8BitDo SN30 Pro+ and a regular Switch Joy-Con (L) to my Mac via Bluetooth. Both controllers show up in System Preferences as connected.

When I got to the OpenEmu preferences and try to set up the controllers, I am only allowed to use one of the controllers at a time...I can't figure out how to use both controllers at the same time (one controller for player 1 and one controller for player 2). I've tried searching GitHub for help, but I can't find any solutions.

I'm using a 2020 MacBook Pro (Intel) running macOS Monterey (12.6).

**Comment by DestructoDisk** [comment]
Just to confirm, you are selecting "player 2" from the player select drop down menu, and selecting your controller from the bottom drop down menu labeled "input".
![Screenshot 2023-02-18 at 8 05 58 PM](https://user-images.githubusercontent.com/94093921/219921637-ba7340cc-d929-4c38-92d6-78f59f05c0a4.png)

**Comment by gcspoon** [comment]
Yes, I set Player 1 to the Joy-Con and Player 2 to the SN30 Pro+ (which shows up as a Switch Pro Controller). In the Open Emu controller preferences, everything looks correct.....but when I try to start the game, both controllers don't work (I press all the buttons and nothing happens). If I disconnect one of the controllers (e.g. the SN30 Pro+), everything works fine with the Joy-Con and I can use the Joy-Con as a controller. If I try to connect the SN30 Pro+ again, the Joy-Con stops working and both controllers don't work.

I recorded a screencast of what happens: https://drive.google.com/file/d/1NNr26CSoRoCJvDpO3DF7eTyd3jW0A3B5/view?usp=sharing
